CMakeLists.txt

```diff
@@ -243,10 +242,25 @@ if(EXISTS "${PROJECT_SOURCE_DIR}/3rdparty/liblsl/CMakeLists.txt" AND NOT WIN32)
     add_subdirectory( plugins/DataStreamLSL )
 endif()
 
+# build cereal and opendbc before we search for required Cereal plugin libs below
+add_custom_command(
```

Comment:
move this to the plugin's make file

Reply:
The lines below it rely on what this command builds. Want me to removvev FindCereal and always build it?